ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,306 @@
+    fi
+
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Wazuh repository could not be added"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Wazuh manager
+installWazuh() {
+
+
+    logger "Installing the Wazuh manager..."
+    eval "$sys_type install wazuh-manager-${WAZUH_VERSION} -y -q $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Wazuh installation failed"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Elasticsearch
+installElasticsearch() {
+    logger "Installing Open Distro for Elasticsearch..."
+  #  eval "curl https://d3g5vo6xdbdb9a.cloudfront.net/yum/opendistroforelasticsearch-artifacts.repo -o /etc/yum.repos.d/opendistroforelasticsearch-artifacts.repo"
+    eval "$sys_type install opendistroforelasticsearch-${OPENDISTRO_VERSION} -y -q $debug"
+
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Elasticsearch installation failed"
+        exit 1;
+    else
+        logger "Done"
+
+        logger "Configuring Elasticsearch..."
+
+        eval "curl -so /etc/elasticsearch/elasticsearch.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/7.x/elasticsearch_all_in_one.yml --max-time 300"
+        eval "curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/roles/roles.yml --max-time 300 "
+        eval "curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles_mapping.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/roles/roles_mapping.yml --max-time 300 "
+        eval "curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/roles/internal_users.yml --max-time 300 "
+        eval "rm /etc/elasticsearch/esnode-key.pem /etc/elasticsearch/esnode.pem /etc/elasticsearch/kirk-key.pem /etc/elasticsearch/kirk.pem /etc/elasticsearch/root-ca.pem -f "
+        eval "mkdir /etc/elasticsearch/certs "
+        eval "cd /etc/elasticsearch/certs "
+        eval "curl -so /etc/elasticsearch/certs/search-guard-tlstool-1.8.zip https://maven.search-guard.com/search-guard-tlstool/1.8/search-guard-tlstool-1.8.zip --max-time 300 "
+        eval "unzip search-guard-tlstool-1.8.zip -d searchguard "
+        eval "curl -so /etc/elasticsearch/certs/searchguard/search-guard.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/searchguard/search-guard-aio.yml --max-time 300 "
+        eval "chmod +x searchguard/tools/sgtlstool.sh "
+        eval "./searchguard/tools/sgtlstool.sh -c ./searchguard/search-guard.yml -ca -crt -t /etc/elasticsearch/certs/ $debug "
+        if [  "$?" != 0  ]
+        then
```

Comment:
```suggestion
        if [ "$?" != 0 ]; then
```

Reply:
Done in 528feec208a367ba616fb87ea4237096d0681494